conf/mesos-site.xml

```diff
@@ -89,6 +89,21 @@
     <value>3</value>
   </property>
 
+  <property>
+    <name>mesos.hdfs.jre-url</name>
+    <value>https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz</value>
```

Comment:
Not sure I understand, but the ones you have listed are appropriate defaults and should be set in HDFSFrameworkConf not in this file (although the user can override them here). https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz, jre1.7.0_76, /opt/mesosphere/lib seems like reasonable defaults